css/css-backgrounds/parsing/border-image-valid.html

```diff
@@ -14,35 +14,46 @@
 <script>
 // Blink and WebKit fail all these tests by returning an empty string as the value of border-image.
 
-test_valid_value("border-image", 'url("http://www.example.com/") 1 2 3 4 fill');
-
-test_valid_value("border-image", "none");
-test_valid_value("border-image", 'url("http://www.example.com/")');
-
-test_valid_value("border-image", "stretch");
-test_valid_value("border-image", "repeat round");
-test_valid_value("border-image", "space space", "space");
-
-test_valid_value("border-image", "1");
-test_valid_value("border-image", "1 2% 3 4%");
-test_valid_value("border-image", "1 2% 3 4% fill");
-test_valid_value("border-image", "fill 1 2% 3 4%", "1 2% 3 4% fill");
-
-test_valid_value("border-image", "1 / 1px");
-test_valid_value("border-image", "1 2% 3 4% / 2%");
-test_valid_value("border-image", "1 2% 3 4% fill / 3");
-test_valid_value("border-image", "fill 1 2% 3 4% / auto", "1 2% 3 4% fill / auto");
-test_valid_value("border-image", "1 / 1px 2% 3 auto");
-
-test_valid_value("border-image", "1 / / 1px");
-test_valid_value("border-image", "1 2% 3 4% / / 2");
-test_valid_value("border-image", "1 2% 3 4% fill / / 1px 2 3px 4");
-
-test_valid_value("border-image", "1 / 1px / 1px");
-test_valid_value("border-image", "1 2% 3 4% / 2% / 2");
-test_valid_value("border-image", "1 2% 3 4% fill / 3 / 1px 2 3px 4");
-test_valid_value("border-image", "1 / auto / 1px");
-test_valid_value("border-image", "1 2% 3 4% / 1px 2% 3 auto / 2");
+// "none" in Edge, "none 100% / 1 / 0 stretch" in Firefox.
```

Comment:
What about WK / Blink? I think Edge would be ideally right here.

Reply:
Blink and WebKit fail all these tests by returning an empty string as the value of border-image.